frontend/pages/hosts/ManageHostsPage/_styles.scss

```diff
@@ -300,4 +300,14 @@
       top: -2px;
     }
   }
+
+  .mdm_enrollment_status__header,
+  .mdm_server_url__header {
+    .last-col-header-with-tip {
+      width: 170px;
+    }
+    .component__tooltip-wrapper__tip-text {
+      left: -156px; // Wide tooltip needs to be shifted left (edge case example when it's second to last column)
```

Comment:
I think this is supposed to compensate for above? Maybe needs just a couple more pixels